src/install/aim/agent-lang/python-config.mdx

```diff
@@ -0,0 +1,71 @@
+---
+componentType: default
+headingText: Configure the Python agent
+---
+
+You installed the Python agent in the last step, but you still need to make configurations so it recognizes and collects AI-specific data. For AI data, you need to enable the `ai_monitoring` and `ai_monitoring.streaming` flags. Both prompt the Python agent to expose AI data within your app data. 
+
+1. We recommend applying these configurations from `newrelic.ini` config file. If you have a more complex set up with multiple environments, you can configure with environment variables.
+
+    <CollapserGroup>
+        <Collapser
+            id="config-file"
+            title="Enable with config file"
+        >
+
+    ```yaml
+    ai_monitoring.enabled = true
+    ai_monitoring.streaming = true
+    ```   
+        </Collapser>
+        <Collapser
+            id="config-enviro"
+            title="Enable with environment variables"
+        >
+
+    ```bash
+    NEW_RELIC_AI_MONITORING_ENABLED=true
+    NEW_RELIC_AI_MONITORING_STREAMING=true
+    ```
+
+        </Collapser>
+    </CollapserGroup>
+
+3. One instance of the Python agent sends a maximum of 100,000 machine learning (ML) events per minute. Once the agent hits that maximum, it samples your ML events, giving a general representation of events across the harvest cycle. 
+
+    You can adjust this value to ELEPHANT.
+
+        <CollapserGroup>
+            <Collapser
+                id="config-file"
+                title="Enable with config file"
+            >
+
+        ```yml
+        event_harvest_config.harvest_limits.ml_event_data = <YOUR_VALUE>
+        ```
+            </Collapser>
+            <Collapser
+                id="config-enviro"
+                title="Enable with environment variables"
+            >
+
+        ```bash
+        NEW_RELIC_ML_INSIGHTS_EVENTS_MAX_SAMPLES_STORED = <YOUR_VALUE>
+        ```
+
+            </Collapser>
+        </CollapserGroup>
+
+4. These configurations adjust how the agent captures and exposes your AI data. For example, if `distributed_tracing.enabled` is set to `false`, then your AI data would be limited to ELEPHANT. Adjusting the quantity of sampled events or spans can ELEPHANT.  
+
+    * `distributed_tracing.enabled = true`: Enables distributed tracing
```

Comment:
Same question as with Node. This seems like it *needs* to be enabled with the ai_monitoring feature flag. Should we move this to the 'necessary' configurations with ai_monitoring.enabled and ai_monitoring.streaming? @RyanKadri

Reply:
Not opposed to moving it to the necessary section. I think our current stance is that we'll show as much as we can without it but you'll lose out on a fair bit of functionality. So it's probably fine to just call it required